test/extended/builds/start_build.go

```diff
@@ -0,0 +1,52 @@
+package builds
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("cli: start-build", func() {
```

Comment:
@deads2k I put it into cli bucket because I don't want to run all build tests, just exercise the behavior of the CLI command... I can add them to build: as well, no strong opinions about that.